diff -urN libgrss-0.4.0/libgrss.pc.in libgrss-0.4.0.new/libgrss.pc.in
--- libgrss-0.4.0/libgrss.pc.in	2010-04-21 16:15:02.000000000 +0200
+++ libgrss-0.4.0.new/libgrss.pc.in	2011-03-15 20:30:05.400297719 +0100
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
-includedir=${exec_prefix}/include
+libdir=@libdir@
+includedir=@includedir@
 
 Name: libgrss
 Description: GObject RSS handling library
